opae.spec.fedora

```diff
@@ -95,7 +96,7 @@ Additional OPAE tools
        -DOPAE_BUILD_FPGABIST=ON 
 
 %if 0%{?rhel}
-  %make_build
+  %cmake_build
```

Comment:
This was per Tom.  Why are we changing it?

Reply:
changes from  %make_build  to %cmake_build because of genearte rpm fails on RHEL 9.0 version